modules/telco-troubleshooting-mco-purpose.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/day_2_core_cnf_clusters/troubleshooting/telco-troubleshooting-mco.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="telco-troubleshooting-mco-purpose_{context}"]
+= Purpose of the Machine Config Operator
+
+The Machine Config Operator (MCO) manages and applies configuration and updates of {op-system-first} and container runtime, including everything between the kernel and kubelet.
+Managing {op-system} is important since most telecommunications companies run on bare-metal hardware and use some sort of hardware accelerator or kernel modification.
+Applying machine configuration to {op-system} manually can cause problems because the Machine Config Operator monitors each node and what is applied to them.
```

Comment:
```suggestion
Applying machine configuration to {op-system} manually can cause problems because the Machine Config Operator monitors each node and what is applied to it.
```
"each node" is singular